### PR TITLE
ci: create documentation example cron job

### PR DESCRIPTION
**Pull request by metaclips**
### Proposed Changes
Test documentation examples hourly

### CODE REVIEW

File: .github/workflows/documentation_examples_check.yml [hunk 1]
```diff
@@ -1,5 +1,14 @@
 name: documentation_examples_check
-on: push
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *'
```
Comment (mrinalwadhwa) [inline .github/workflows/documentation_examples_check.yml:5]: Lets change this to [every two hours](https://cron.help/#0_*/2_*_*_*)


```
0 */2 * * *
```

File: .github/workflows/documentation_examples_check.yml [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
```
Comment (mrinalwadhwa) [inline .github/workflows/documentation_examples_check.yml:11]: I think - on push should be sufficient, let's remove the pull request trigger. In the past when I've tried keeping both triggers we get unnecessary runs

**Review by mrinalwadhwa — CHANGES_REQUESTED** [review]
Thank you @metaclips. I left a couple of small change requests.

File: .github/workflows/documentation_examples_check.yml (hunk 1 above)
Comment (metaclips) [reply .github/workflows/documentation_examples_check.yml]: Thanks

File: .github/workflows/documentation_examples_check.yml (hunk 2 above)
Comment (metaclips) [reply .github/workflows/documentation_examples_check.yml]: Fixed

**Comment by jared-s** [comment]
Merged as https://github.com/ockam-network/ockam/pull/1730

**Pull request closed without merging**